clang/include/clang/Basic/DiagnosticParseKinds.td

```diff
@@ -1581,8 +1581,9 @@ def err_omp_unexpected_append_op : Error<
   "unexpected operation specified in 'append_args' clause, expected 'interop'">;
 def err_omp_unexpected_execution_modifier : Error<
   "unexpected 'execution' modifier in non-executable context">;
-def err_omp_unknown_adjust_args_op : Error<
-  "incorrect adjust_args type, expected 'need_device_ptr' or 'nothing'">;
+def err_omp_unknown_adjust_args_op
+    : Error<"incorrect adjust_args type, expected 'need_device_ptr'%select{|, "
```

Comment:
```suggestion
    : Error<"incorrect 'adjust_args' type, expected 'need_device_ptr'%select{|, "
```
I see this is an existing issue, but we usually put single quotes around syntactic constructs.